src/App.css

```diff
@@ -1,3 +1,320 @@
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  margin: 0.188rem 0;
+}
+
+.book-author {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: var(--pale-blue);
+  margin: 0.25rem 0 1.313rem;
+}
+
+.book-buttons button {
+  cursor: pointer;
+  font-family: inherit;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: var(--pale-blue);
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  padding: 0;
+  padding-right: 0.9rem;
+  border: none;
+  border-right: 1px solid #e8e8e8;
+  outline: none;
+  background-color: transparent;
+}
+
+.book-update button {
+  cursor: pointer;
+  width: 11.5rem;
+  height: 2.063rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--primary-color);
+  color: #fff;
+  letter-spacing: 0.5px;
+  font-size: 0.813rem;
+  font-weight: 300;
+  border: none;
+}
+
+.book-buttons button:last-of-type {
+  border-right: none;
+}
+
+.book-progress {
+  display: flex;
+  margin: 0 30px;
+}
+
+.circle-wrap {
+  width: 60px;
+  height: 60px;
+  background: #fff;
+  border-radius: 50%;
+  border: 2px solid #e8e8e8;
+  margin-right: 10px;
+}
+
+.mask .fill {
+  clip: rect(0, 30px, 60px, 0);
+  background-color: var(--primary-color);
+}
+
+.mask.full,
+.circle .fill {
+  animation: fill ease-in-out 3s;
+  transform: rotate(160deg);
+}
+
+.circle-wrap .circle .mask,
+.circle-wrap .circle .fill {
+  width: 60px;
+  height: 60px;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.circle-wrap .circle .mask {
+  clip: rect(0, 60px, 60px, 30px);
+}
+
+@keyframes fill {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(160deg);
+  }
+}
+
+.circle-wrap .inside-circle {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  margin-top: 5px;
+  margin-left: 5px;
+  background: #fff;
+  z-index: 1;
+}
+
+.percentage {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  margin-bottom: 0;
+  margin-top: 10px;
+}
+
+.completed {
+  margin-top: 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  opacity: 0.5;
+}
+
+.current-chap {
+  font-size: 0.813rem;
+  font-weight: 300;
+  opacity: 0.5;
+  margin: 0;
+}
+
+.chap-num {
+  margin-top: 0.438rem;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+}
+
+.book-update {
+  border-left: 1px solid #e8e8e8;
+  padding-left: 3.688rem;
+}
+
+hr {
+  margin: 2.5rem 0.063rem 1.813rem 0;
+  border: 1px solid #e8e8e8;
+}
+
+#form-container h2 {
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+#form-container input {
+  width: 25%;
+  margin: 1.188rem 2.125rem 0 0;
+  padding: 8px 15px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  color: #888;
+  font-size: 1rem;
+  outline: none;
+  box-sizing: border-box;
+}
+
+#add-book {
+  cursor: pointer;
+  width: 11.5rem;
+  height: 2.4rem;
+  border-radius: 3px;
+  background-color: var(--primary-color);
+  font-family: inherit;
+  font-size: 0.813rem;
```

Comment:
[OPTIONAL] It would be nicer to align the button next to the form inputs so your styling matches the one provided in Zeplin.